scriptRunner/src/keybase.js

```diff
@@ -28,7 +28,7 @@ module.exports.getKeybaseImages = async () => {
             return {
               name,
               operator_address,
-              picture: result.list[0].keybase.picture_url
+              picture: result.list[0].keybase.picture_url.replace('http', 'https')
```

Comment:
What happens with already stored images?

Reply:
I fixed them with this Postgres command: 
`update polkadot.validatorprofiles  set picture = replace(picture, 'http://', 'https://');`

Reply:
Perfecto